src/services/dashboards/dashboard-detail/DashboardDetailPage.vue

```diff
@@ -31,15 +31,18 @@
                 <dashboard-control-buttons @update:visible-clone-modal="handleVisibleCloneModal" />
             </template>
         </p-page-title>
-        <div class="flex justify-between mt-4">
+        <div class="filter-box">
             <dashboard-labels :label-list="state.labelList" />
             <dashboard-toolset />
         </div>
-        <p-divider />
-        <dashboard-refresher :interval="state.refreshInterval"
-                             :loading="state.loading"
-                             @refresh="handleRefresh"
-        />
+        <p-divider class="divider" />
+        <div class="filter-box">
+            <div>filters</div>
```

Comment:
todo: translation!

Reply:
This is an area that needs to be developed later! 
thx check!